docs/README.md

```diff
@@ -508,6 +508,7 @@ Protocol is published to NPM under the `@mozilla-protocol/core` namespace/packag
 1. Before you start make sure the project's [CHANGELOG.md][changelog] is up to date.
 2. Update the package `version` number in [src/assets/package/package.json][package.json] (use [Semantic Versioning][semver] to determine what the new version number should be).
 3. Update the package README [assets/package/README.md][readme].
+4. Run `npm install` to update the package-lock.json file.
```

Comment:
Technically this shouldn't be required, since running `npm test` will kick off a bunch of pre execution scripts that include dependency installation. That said, I don't think a little redundancy hurts to err on the safe side 👍